tests/dragonfly/snapshot_test.py

```diff
@@ -639,3 +639,34 @@ async def check_flag(key, flag):
 
     await check_flag("key1", 2)
     await check_flag("key2", 123456)
+
+
+@pytest.mark.parametrize("format", FILE_FORMATS)
+@dfly_args({**BASIC_ARGS})
+async def test_infomemory_for_stream(df_factory, format: str):
+    dbfilename = f"dump_{tmp_file_name()}"
+    instance = df_factory.create(dbfilename=dbfilename)
+    instance.start()
+    async_client = instance.client()
+    await async_client.execute_command("XADD test 1 var val1")
+    await async_client.execute_command("XADD test 2 var val2")
+
+    async def assert_stream_mem_not_zero():
+        res = await async_client.execute_command("INFO MEMORY")
+
+        for line in res.splitlines():
+            if line.startswith("type_used_memory_stream"):
```

Comment:
I would check that after adding stream items type_used_memory_stream is > 0, save it to a var and after loading the data again check the value of type_used_memory_stream is the same

Reply:
for simple xadd we do a preallocation so the numbers will be different

Reply:
ok then lets not check that they are the same value but make sure both > 0